source/administration/config-settings.rst

```diff
@@ -3135,13 +3135,23 @@ Enable Developer Mode
 
 Allow untrusted internal connections to
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-This setting allows system administrators to limit the addresses that user-created webhooks and slash commands have access to. This will prevent them from potentially gaining access to network resources via Mattermost that they do not have access to themselves.
+This setting limits the ability for the Mattermost server to make untrusted requests within its local network. A request is considered "untrusted" when it's made on behalf of a client. The following features make untrusted requests and are affected by this setting:
 
-If a user needs to develop an integration locally on their development workstation, use this setting to specify the domain, IP address or CIDR notation for those machines. This will allow users to host the app they're developing on their workstation. **Not recommended for use in production**, since this can allow a user to extract confidential data from your server or internal network.
+- Integrations using webhooks, slash commands, or message actions. This prevents them from requesting endpoints within the local network.
+- Link previews. When a link to a local network address is posted in a chat message, this prevents a link preview from being displayed.
+- The `local image proxy <image-proxy.html#local-image-proxy>`__. If the local image proxy is enabled, images located on the local network cannot be used by integrations or posted in chat messages.
 
-By default, user-supplied URLs such as those used for Open Graph metadata, webhooks or slash commands will not be allowed to connect to reserved IP addresses including loopback or link-local addresses used for internal networks. Push notification and OAuth 2.0 and other URLs entered via the System Console are trusted and not affected by this setting.
+Requests that can only be configured by admins are considered trusted and will not be affected by this setting. Trusted URLs include ones used for OAuth login or sending push notifications.
 
-Separate two or more domains with spaces instead of commas, for example: ``webhooks.internal.example.com 127.0.0.1 10.0.16.0/28``.
+.. warning:: This setting is intended to prevent users located outside your local network from using the Mattermost server to request confidential data from inside your network. Care should be used when configuring this setting to prevent unintended access to your local network.
+
+Some examples of when you may want to modify this setting include:
```

Comment:
Are there specific examples we can share on how an admin might want to modify the setting for these cases?

Reply:
These seem pretty specific to me. Are you looking for something more like "If you have X plugin installed on Y server, add Y to this list?"

Reply:
> I struggled to get that exactly since it depends very much on how their network is set up

Got it, if that's the case, we can keep the examples as-is. We can always update later, if admins are looking for more specific examples.